Detectors/TRD/calibration/src/CalibratorVdExB.cxx

```diff
@@ -126,6 +126,18 @@ void CalibratorVdExB::initProcessing()
   opt.SetTolerance(.001);
   mFitter.Config().SetMinimizerOptions(opt);
 
+  // Set tree branches
+  if (mOutTree) {
+    mOutTree->Branch("lorentzAngle", &mFitFunctor.laPreCorr);
+    mOutTree->Branch("vDrift", &mFitFunctor.vdPreCorr);
+    //mOutTree->Branch("residuals", &mFitFunctor.profiles);
+  }
+
+  // Retrieve configuration values
+  auto& params = TRDCalibParams::Instance();
+  mMinEntriesChamber = params.minEntriesChamber;
+  mMinEntriesTotal = params.minEntriesTotal;
```

Comment:
I initial put this in the 'wrong' function. Programmatically it did matter and the result would have been the same.
But putting this here is saner.